docs/components/modeler/web-modeler/milestones.md

```diff
@@ -1,50 +1,158 @@
 
-When dragging and dropping a file into the diagram view, or when using the **Replace via upload** option under the breadcrumb menu, a new milestone is created automatically.
+- Select **Show milestone history** from the breadcrumb menu.
 
-#### Bulk milestone creation
+  ![Milestone history screen showing the History button](img/milestones/web-modeler-milestone-show-history-via-breadcrumb-highlight.png)
 
-A [process application](/components/modeler/web-modeler/process-applications.md) is a special type of folder in Web Modeler that allows you to work on a set of related files and
-[deploy](/components/modeler/web-modeler/process-applications.md#deploy-and-run-a-process-application) them together in a single bundle with just one click. This reduces the risk of having a broken deployment at runtime and makes it more convenient to deploy related files.
+## Create a milestone
 
-When creating a milestone on a main process of a process application, milestones are created for all other assets in the application to make it easier to track or roll back changes.
+You can create a new milestone either from your diagram or the milestone history.
+
+- From your diagram, select **Create milestone** from the breadcrumb menu.
+
+  ![milestones create via icon](img/milestones/web-modeler-milestone-create-via-breadcrumb-highlight.png)
+
+- From the milestone history, hover over the the latest version in the **Milestones** panel and select **Create a new milestone**.
+
+  ![milestones create via icon](img/milestones/web-modeler-milestone-create-via-icon-highlight.png)
+
+:::note
+
+A new milestone is also automatically created when dragging and dropping a file into the diagram view, or when you select the **Replace via upload** option from the breadcrumb menu.
+
+:::
+
+### Process application bulk milestone creation
+
+A [process application](/components/modeler/web-modeler/process-applications.md) is a special type of Web Modeler folder that allows you to work on a set of related files and
+[deploy](/components/modeler/web-modeler/process-applications.md#deploy-and-run-a-process-application) them as a single bundle with just one click. This reduces the risk of having a broken deployment at runtime, and makes it more convenient to deploy related files.
+
+If you create a milestone on the main process of a process application, milestones are created for all other assets in the application, to make it easier to track or revert changes.
 
 :::note
 Milestones of resources belonging to a process application are tied to the main process and cannot be modified.
 :::
 
-### Restoring milestones
+## Compare milestones
+
+You can compare the change history between two milestones, either visually as a diagram or as code in an XML diff layout.
+
+1. Open the milestone history for your diagram.
+1. Ensure that the **Show changes** toggle in the **Changes** panel is turned on.
```

Comment:
🔧 The toggle is present in the sidebar, not specifically inside the changes panel.

Reply:
@nithinssabu Can I just check here: For me the toggle is in the Changes panel on the side?

![image](https://github.com/user-attachments/assets/5cd9e0f4-2de2-454c-92fd-7d700b6b796c)

Reply:
Changes made (sidebar)